cmd/internal/ecs/add.go

```diff
@@ -85,20 +85,21 @@ type AddWorkflow struct {
 
 const (
 	// Tag to use for objects created by the Akita CLI
-	akitaCreationTagKey       = "akita.software:created_by"
-	akitaCreationTagValue     = "Akita Software ECS integration"
-	akitaModificationTagKey   = "akita.software:modified_by"
-	akitaModificationTagValue = "Akita Software ECS integration"
+	akitaCreationTagKey       = "postman.software:created_by"
+	akitaCreationTagValue     = "Postman Software ECS integration"
+	akitaModificationTagKey   = "postman.software:modified_by"
+	akitaModificationTagValue = "Postman Software ECS integration"
 
 	// Separate AWS secrets for the key ID and key secret
 	// TODO: make these configurable
-	akitaSecretPrefix    = "akita.software/"
+	akitaSecretPrefix    = "postman.software/"
```

Comment:
Same thing here, let's use either a real domain name or something obviously not a domain name so somebody else can't register it.